mkdocs.yml

```diff
@@ -1,7 +1,30 @@
-site_name: PrairieLearn
-site_url: https://github.com/PrairieLearn/PrairieLearn
+site_name: PrairieLearn Docs
+site_url: https://prairielearn.readthedocs.io
 repo_url: https://github.com/PrairieLearn/PrairieLearn
-theme: readthedocs
+theme:
+  name: material
+  logo: assets/PL-logo.svg
+  favicon: assets/favicon.ico
+  icon:
+    repo: fontawesome/brands/github
+    edit: material/pencil
+  palette:
+    scheme: default
+    primary: custom
+  features:
+    - navigation.tabs
+    - navigation.tabs.sticky
+    - navigation.path
+    - navigation.instant
+    - navigation.instant.prefetch
+    - navigation.instant.progress
+    - navigation.tracking
+    - search.suggest
+    - content.action.edit
+plugins:
+  - search
+  - git-revision-date-localized:
+      enable_creation_date: true
```

Comment:
This adds ~8s to rebuilds for me locally, which is a bit annoying. You asked elsewhere whether we have the full history when the docs are built: we don't. Maybe we just remove this plugin? Some of the docs I'm most familiar with (React, GitHub) don't show any dates in their docs and that's never bothered me 🤷